carbon_cassandra_plugin/carbon_cassandra_db.py

```diff
@@ -364,7 +365,13 @@ def _metric_lookup(query_split, query_re, level, child_nodes):
       # Also, "root" will not be a literal match against most metrics using RE.
       if query == "root" or not _query_contains_wildcards(query):
         child_nodes = _get(query)
+      elif query[0] == '*':
```

Comment:
`_query_contains_wildcards` should cover this case, right? If not then it seems like there's something missing in that function.